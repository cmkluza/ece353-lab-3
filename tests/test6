Tests ADD, MUL, and SUB, with no errors, with m = 2, n = c = 1
./sim-mips -b 2 1 1 ./tests/asm/test2.s ./tests/out/test6.out
./tests/sols/test2.sol