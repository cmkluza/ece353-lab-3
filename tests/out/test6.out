program name: ./tests/asm/test2.s
stage utilization: 0.653846  0.615385  0.730769  0.000000  0.615385 
register values 0  0  0  0  0  0  0  1  2  3  255  256  32767  0  0  3  6  65534  4  6  512  -262140  1  -254  1  0  0  0  0  0  0  68
